test/helpers/expectEvent.js

```diff
@@ -16,16 +19,22 @@ function inLogs (logs, eventName, eventArgs = {}) {
   return event;
 }
 
-async function inTransaction (tx, eventName, eventArgs = {}) {
-  const { logs } = await tx;
+async function inConstruction (contract, eventName, eventArgs = {}) {
+  return inTransaction(contract.transactionHash, contract.constructor, eventName, eventArgs);
+}
+
+async function inTransaction (txHash, emitter, eventName, eventArgs = {}) {
```

Comment:
We really should start documenting these functions. For example that `emitter` is a contract class.

Reply:
Agreed, though that's a larger task that needs to be more properly defined (e.g. do we want to be able to autogenerate a simple docsite from those comments?). The main reason why I haven't yet sat down to do it is that we're iterating over the API somewhat fast, and don't want to have to re-write the docs daily.